src/App/App.js

```diff
@@ -4,12 +4,17 @@ import { Router, Route } from 'react-router-dom';
 import { connect } from 'react-redux';
 /* THIRD PARTY IMPORTS */
 import CssBaseline from '@material-ui/core/CssBaseline';
+import { makeStyles } from '@material-ui/core/styles';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
 /* LOCAL IMPORTS */
 import { history } from '../_helpers';
 import { alertActions } from '../_actions';
 import { PrivateRoute } from '../_components';
 import { AboutUsPage } from '../AboutUsPage';
 import { LoginPage } from '../LoginPage';
+import DiscreteSlider from '../_components/vSlider';
```

Comment:
Take out this line

Reply:
The import tabs line?

Reply:
OKay, the react-scripts line. I believ it came in because I installed react-scripts on my system.